README.md

```diff
@@ -4,14 +4,12 @@
 
 ![HoneyGopher](.github/images/cilium-ebpf.png)
 
-eBPF is a pure Go library that provides utilities for loading, compiling, and
+ebpf-go is a pure Go library that provides utilities for loading, compiling, and
```

Comment:
Clear as mud? ;D https://pkg.go.dev/github.com/dropbox/goebpf Referring to it as `ebpf-go` when speaking about it would work for me, but I don't want to rename the package `ebpf-go` since that forces import renaming.

Reply:
Of course not, no import renaming. :)